### ISSUE TITLE
🐛[BUG]

### ISSUE THREAD

**Issue by Gopisuresh-Akula** [ant-design/ant-design-charts#919]
Hi I facing an issue with charts please find  attach the image response as soon as possible 

![Uploading imag
![jj](https://user-images.githubusercontent.com/42737147/136886808-8b51b6b0-2d5f-4099-9048-32a9e33f293a.png)
e.png…]()

**Comment by Gopisuresh-Akula** [comment]
yeah deleted yarn lock --> yarn installl but issue not resolved more over I'm using webpack

**Comment by Gopisuresh-Akula** [comment]
![jj](https://user-images.githubusercontent.com/42737147/136887664-bb64275c-fe11-491f-8f34-b2c94becbc18.png)

**Comment by aniruddhashevle** [comment]
@Gopisuresh-Akula : Can you please tell us which versions of ant-design-charts, webpack, node, npm your using?
@lxfu1 @hitf2 @Yanyan-Wang @baizn : Can you please help us guide on this?

**Comment by Gopisuresh-Akula** [comment]
@lxfu1  @aniruddhashevle  "@ant-design/charts": "^1.2.13",   
 "webpack": "3.8.1",
    "webpack-dev-server": "2.9.4",
    "webpack-manifest-plugin": "1.3.2", these my vesrion
node:v14.17.6
npm:6.14.15
yarn:1.22.5

**Comment by jamesadhitthana** [comment]
Hi there, I am also facing the exact same issue as above where `ml-matrix` fails to minify while running `npm run build` or `yarn build`. We have narrowed it down to ant-design-charts as the problem because uninstalling ant-design-charts causes the problem to disappear (however we would like to prevent doing this as much as possible).

Here are the versions requested:
```
"antd": "^3.12.2",
"@ant-design/charts": "^1.2.10",
"webpack": "3.8.1",
node: v14.16.0
npm: 6.14.11
```
Note: I have also tried to update the ant design charts to the newest version however the problem still persists.

Are there any updates on solving this issue?

Thank you.

**Comment by pravins007** [comment]
Hi there, I'm also facing the same issue while npm run build 

node_modules\ml-matrix\src\symmetricMatrix.js: Unexpected character '#'

are there any updated on resolving this issue
